caput/config.py

```diff
@@ -514,16 +514,21 @@ def _prop(val):
         if val is None:
             return None
 
+        if issubclass(val, fileformats.FileFormat):
+            return val
+
         if not isinstance(val, str):
             CaputConfigError(
```

Comment:
While we're editing this file, we should probably fix:
```suggestion
            raise CaputConfigError(
```

Reply:
Good catch! I was trying to figure out why it failed at the call to `val.lower()` and not here, and I somehow completely missed that this wasn't actually being raised!